hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/TenantStatsManagement.java

```diff
@@ -16,19 +16,17 @@
  * Management service for statistics of a single tenant.
  *
  */
-@FunctionalInterface
 public interface TenantStatsManagement {
 
     /**
-     * Service for stats of a single tenant. Opens a new transaction and as a
-     * result can an be used for multiple tenants, i.e. to allow in one session
-     * to collect data of all tenants in the system.
+     * Service for stats of a single tenant.
      *
-     * @param tenant
-     *            to collect for
      * @return collected statistics
      */
-    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_SYSTEM_ADMIN)
-    TenantUsage getStatsOfTenant(String tenant);
+    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_REPOSITORY + SpringEvalExpressions.HAS_AUTH_OR
```

Comment:
Is there a reason why we need so many different permissions for this? Need some clarification. I'd like to add in hawkBit that system code is generally allowed to call management APIs.